src/dune_rules/assets.ml

```diff
@@ -0,0 +1,77 @@
+let jbuild_plugin_ml =
```

Comment:
This file looks duplicated now (it's both in engine and in rules).

Reply:
Hopefully fixed